compiler/ir/ir.psi2ir/src/org/jetbrains/kotlin/psi2ir/generators/BranchingExpressionGenerator.kt

```diff
@@ -77,7 +77,7 @@ class BranchingExpressionGenerator(statementGenerator: StatementGenerator) : Sta
         if (irBranches.size == 1) {
```

Comment:
@dnpetrov Shoudn't we use IF origin  for irBranches.size > 1 below:
```
val irWhen = IrWhenImpl(ktIf.startOffsetSkippingComments, ktIf.endOffset, resultType, IrStatementOrigin.WHEN)
```

@ting-yuan Please also add next test:
```
fun cond() = false

fun bar() {}

fun foo() {
    if (cond()) {
        bar()
    } else if (true) {
        bar()
    } else {
        bar()
    }
}
// 1 LINENUMBER 6
// 1 LINENUMBER 8
```

Reply:
Similiar tests with 'false' would be also appreciated